src/components/ItemList/ItemRow.scss

```diff
@@ -0,0 +1,15 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.ba-ItemRow-name {
+    color: $bdl-gray;
+    font-size: 13px;
+    line-height: 15px;
+    letter-spacing: .3px;
+}
+
+.ba-ItemRow-email {
+    color: $bdl-gray-62;
+    font-size: 11px;
+    line-height: 15px;
+    letter-spacing: .25px;
```

Comment:
Can you check with design on whether this difference was intentional? It's off standard and the difference is very small. Maybe we can just use the default `.3px`.